src/__test__/utils/cellSets/getContainingCellSetsProperties.test.js

```diff
@@ -4,10 +4,10 @@ import mockCellSets from 'utils/tests/mockStores/cellSets';
 describe('get cell class properties tests', () => {
   const { cellSets } = mockCellSets();
   it('returns information for a passed cellId', () => {
-    // doing it as a string to make sure it still works (sometimes a string is passed from vitessce)
```

Comment:
So having `cellId` as a string doesn't work?

Reply:
before the function would convert it to int - now its changed to expect an int so this string - int conversion is done only in the places that receive string